cbdr/cortex
# COR-672: Enhance Click to Edit Index

**Purpose:** To make rows clickable and remove creator avatars from index


**JIRA:** [/browse/COR-672](https://cb-content-enablement.atlassian.net/browse/COR-672)


**Changes:**
* Changes to setup
  * must point `cortex-plugins-core` to branch `COR-672/Enhance-Click-To-Edit-Index`
  * run `bundle exec rake cortex:core:db:reseed`

* Architectural changes
  * change render method for the Creator column to be `author_email`

* Migrations
  * `bundle exec rake cortex:core:db:reseed`
  
* Library changes
  * must point `cortex-plugins-core` to branch `COR-672/Enhance-Click-To-Edit-Index`

* Side effects
  * n/a

**Screenshots**
* Before
![image-2017-02-16-09-45-53-004](https://cloud.githubusercontent.com/assets/10361180/24718399/7ddc7f32-19fb-11e7-8d87-3c55c43c7d88.png)

* After
![cortex administration media 2017-04-05 12-28-35](https://cloud.githubusercontent.com/assets/10361180/24718403/81d16788-19fb-11e7-80e5-6ed44344a619.png)


**QA Links:**

[http://web.cortex-2.development.c66.me](http://web.cortex-2.development.c66.me)
**How to Verify These Changes**
* Specific pages to visit
  * blog or media index

* Steps to take
  * click on rows and check hover state

* Responsive considerations
  * n/a


**Relevant PRs/Dependencies:**

[Cortex-Plugins-Core](https://github.com/cortex-cms/cortex-plugins-core/pull/55/files)
**Additional Information**

You may notice that the whole row is not clickable, this is because `tr` and `td` tags cannot be wrapped in a tags since it's invalid HTML. So I listed out a few options to resolve this if this solution is deemed not optimal: [jira Permalink](https://cb-content-enablement.atlassian.net/browse/COR-672?focusedCommentId=28310&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-28310)
